data/data/aws/vpc/vpc.tf

```diff
@@ -1,6 +1,8 @@
 locals {
   new_worker_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,1)}"
   new_master_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,0)}"
+
+  cluster_domain = "${var.cluster_name}.${var.base_domain}"
```

Comment:
This module could be adjusted to take the unsplit cluster domain and use it where it currently uses `var.cluster_name` in tag values and resource names.  That would address #762, although we might have to use the length-limited cluster ID for the load-balancer names or somehow introduce uniqueness there.  I've just filed #1170 making cluster-names in `metadata.json` libvirt-specific, so this resource naming is the last #762 blocker.

Reply:
https://github.com/openshift/installer/issues/762 @blrm was looking at finding the resources that would conflict if we try to use same cluster name for same base domain. I remember him mentioning that it was more than this module.

@blrm do you have some notes/finding that can provide more details on what might collide in #762  :innocent:

Reply:
> @blrm was looking at finding the resources that would conflict if we try to use same cluster name for same base domain.

I'd guess:

```console
$ git describe
unreleased-master-177-g4907cba
$ git grep ' name *= .*cluster_name' data/data/aws | cat
data/data/aws/bootstrap/main.tf:  name = "${var.cluster_name}-bootstrap-profile"
data/data/aws/bootstrap/main.tf:  name  = "${var.cluster_name}-bootstrap-role"
data/data/aws/bootstrap/main.tf:  name  = "${var.cluster_name}-bootstrap-policy"
data/data/aws/iam/main.tf:  name = "${var.cluster_name}-worker-profile"
data/data/aws/iam/main.tf:  name  = "${var.cluster_name}-worker-role"
data/data/aws/iam/main.tf:  name  = "${var.cluster_name}_worker_policy"
data/data/aws/main.tf:  name    = "${var.cluster_name}-etcd-${count.index}"
data/data/aws/main.tf:  name    = "_etcd-server-ssl._tcp.${var.cluster_name}"
data/data/aws/master/main.tf:  name = "${var.cluster_name}-master-profile"
data/data/aws/master/main.tf:  name  = "${var.cluster_name}-master-role"
data/data/aws/master/main.tf:  name  = "${var.cluster_name}_master_policy"
data/data/aws/route53/base.tf:  name    = "${var.cluster_name}-api.${var.base_domain}"
data/data/aws/route53/base.tf:  name    = "${var.cluster_name}-api.${var.base_domain}"
data/data/aws/vpc/master-elb.tf:  name                             = "${var.cluster_name}-int"
data/data/aws/vpc/master-elb.tf:  name                             = "${var.cluster_name}-ext"
data/data/aws/vpc/master-elb.tf:  name     = "${var.cluster_name}-api-int"
data/data/aws/vpc/master-elb.tf:  name     = "${var.cluster_name}-api-ext"
data/data/aws/vpc/master-elb.tf:  name     = "${var.cluster_name}-services"
```

but I don't see a problem with removing those in stages.